kolibri/plugins/coach/assets/src/modules/lessonResources/handlers.js

```diff
@@ -267,18 +267,22 @@ function _prepLessonContentPreview(store, classId, lessonId, contentId) {
     getParams: { no_available_filtering: true },
   }).then(
     contentNode => {
-      const assessmentMetadata = contentNode.assessmentmetadata;
       store.commit('lessonSummary/SET_STATE', {
         toolbarRoute: {},
         // only exist if exercises
         workingResources: null,
         resourceCache: cache,
       });
+
       store.commit('lessonSummary/resources/SET_CURRENT_CONTENT_NODE', contentNode);
-      store.commit('lessonSummary/resources/SET_PREVIEW_STATE', {
-        questions: assessmentMetadata.assessment_item_ids,
-        completionData: assessmentMetadata.mastery_model,
-      });
+
+      if (contentNode.assessmentmetadata) {
+        store.commit('lessonSummary/resources/SET_PREVIEW_STATE', {
+          questions: contentNode.assessmentmetadata.assessment_item_ids,
+          completionData: contentNode.assessmentmetadata.mastery_model,
+        });
+      }
```

Comment:
I initially considered just updating the `lessonSummary/resources/SET_CURRENT_CONTENT_NODE` handler to just set the `state.preview` value if/when these values are present since we're already sending it the `contentNode` but went with the path of least resistance here to avoid giving our neglected Vuex modules any false hope of being refactored in the future, for their fated doom is imminent.